<launch>
    <node name="map_segmentation" pkg="task_1" type="map_segmentation.py" output="screen">
        <param name="segmentation_size" type="int" value="1" />
        <param name="min_quad_size" type="int" value="15" />
    </node>
</launch>
